src/admin.ts

```diff
@@ -1,9 +1,5 @@
 import axios from "axios";
-import {
-  Runnable,
-  VersionedRunnable,
-  UserFunctionsParams,
-} from "./types/runnable";
+import { Module, VersionedModule, UserFunctionsParams } from "./types/module";
```

Comment:
`AvailableFunction` needs to change `fqfn` and `fqfnURI` below.

Reply:
Did the smoke tests pass on your machine? Otherwise, code looks good, other than the small above change :)